.travis.yml

```diff
@@ -31,6 +31,7 @@ jobs:
   - name: "make docs"
     install:
       - pip install -r requirements.txt
+      -pip install -r dev_requirements.txt
```

Comment:
_Very_ small nit-pick here, could you add a space between the `-` and `pip` so it's `- pip install -r dev_requirements.txt`